generic_config_updater/generic_updater.py

```diff
@@ -382,21 +362,27 @@ def get_config_wrapper(self, dry_run):
         else:
             return ConfigWrapper()
 
+    def get_patch_sorter(self, non_strict, ignore_more_list, config_wrapper, patch_wrapper):
+        if non_strict:
+            return NonStrictPatchSorter(ignore_more_list, config_wrapper=config_wrapper, patch_wrapper=patch_wrapper)
```

Comment:
>ignore_more_list [](http://example.com/codeflow?start=40&length=16)

From your implementation, I see there is a dependency between ignore_more_list and non_strict.

However I see they are independent. I feel there is a usage  that all the yang models are available, but we need to ignore yang validation error with existing yang models.

Reply:
This concern will further lead to my comment on command option long names.

Reply:
how do you suggest we make them independent?

non-strict: ignores validation for tables with YANG
ignore-more-...: ignores validation for more tables/fields besides the already ignored by default

Reply:
In the case of all yang models are available we have no problem using the `-n` mode which will do nothing since all tables have yang models already. Then we can use `-i` to ignore more, it is a little inconvenient though.

maybe we can have something like:
```
--non-yang-tables-allowed  -n
--ignore-from-yang-validation -i
```
These flags are independent but if any of them exist they activate the non-strict mode. What do you think?

Reply:
Another idea is:
```
--ignore-path -i    will ignore the give-path from yang validation
--ignore-all-non-yang-tables -n will ignore all tables without yang
```
they are also independent

Reply:
Made the options independent.

The options now are:
```
--n --ignore-non-yang-tables        ignore validation for tables without YANG models
--i --ignore-path                   ignore validation for config specified by given path which is a JsonPointer

```